src/partials/head-meta-twitter.hbs

```diff
@@ -0,0 +1,7 @@
+<!-- START Twitter tags -->
```

Comment:
Please remove these comments, they are also visibe in the source code and IMHO this is absolutely not needed, these tags are pretty much wellknown.